site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -34,14 +34,31 @@ An important configuration to note is the PodDisruptionBudget fields, `agones.ex
 
 ## Deployment Considerations
 
-When `SplitControllerAndExtensions` is enabled, what was previously a single `agones-controller` pod is deployed as `agones-controller` and 2 `agones-extensions` pods. For example: 
+Leader election will automatically be enabled when `SplitControllerAndExtensions` is enabled and `agones.controller.replicas` is > 1. [`agones.controller.replicas`]({{< relref "/docs/Installation/Install Agones/helm.md#configuration" >}}) defaults to 2.
+
+When `SplitControllerAndExtensions` is enabled, what was previously a single `agones-controller` pod is deployed as , possibly 2+, `agones-controller` and 2 `agones-extensions` pods. For example: 
```

Comment:
> `as , possibly 2+,`

just say 2. Both replica settings are changeable, so you don't need to say "possibly 2+". Maybe like this:

```
When `SplitControllerAndExtensions` is enabled, what was previously a single `agones-controller` pod is deployed by default as 2 `agones-controller` and 2 `agones-extensions` pods. For example: